pkg/gardenlet/controller/shoot/care/health.go

```diff
@@ -430,6 +441,37 @@ func (h *Health) checkControlPlane(
 	return &c, nil
 }
 
+// CheckIfDependencyWatchdogProberScaledDownControllers checks if controllers have been scaled down by
+// dependency-watchdog-prober.
+func CheckIfDependencyWatchdogProberScaledDownControllers(ctx context.Context, seedClient client.Client, shootNamespace string) ([]string, error) {
+	var scaledDownDeploymentNames []string
+
+	// Error is ignored since this function never returns an error
+	proberConfig, _ := kubeapiserver.NewDependencyWatchdogProberConfiguration()
```

Comment:
Things might change over time. This comment suggests that there never will be an error here. In this case the function could be changed to not have a second return value. Otherwise having a simply error check does not seem to hurt.